packages/editor/src/components/editor/canvas/Canvas.css

```diff
@@ -4,10 +4,12 @@
   display: flex;
   flex-direction: column;
   margin: var(--size-4);
+  padding: var(--size-2);
   overflow: auto;
-
+  border-radius: var(--border-r2);
```

Comment:
still needed, as you changed it in the styles itself?

Reply:
yes, because this border-radius is used for the rounded edges of the whole canvas:
![grafik](https://github.com/axonivy/form-editor-client/assets/141223521/832c58dc-8062-4b62-a97a-9c3049fcc55b)

Reply:
ah sorry. my bad, code reading is hard :D
I thought you still had the --border-radius variable in usage.